ci_framework/roles/os_net_setup/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+# assiming having a login session with the right namespace already
+- name: get openstack admin credentials from k8s
+  block:
+  - name: Get clouds.yaml configuration
+    ansible.builtin.command:
+      cmd: oc get configmaps openstack-config -o jsonpath={.data}
+    register: clouds_yaml
+    environment:
```

Comment:
you need to specify the KUBECONFIG var here too:
```
TASK [os_net_setup : Get clouds.yaml configuration _raw_params=oc get configmaps openstack-config -o jsonpath={.data}] *********************************************************************
Friday 09 June 2023  00:12:36 +0300 (0:00:00.060)       0:23:17.457 ***********
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["oc", "get", "configmaps", "openstack-config", "-o", "jsonpath={.data}"], "delta": "0:00:00.074515", "end": "2023-06-09 00:12:36.46
8012", "msg": "non-zero return code", "rc": 1, "start": "2023-06-09 00:12:36.393497", "stderr": "error: Missing or incomplete configuration info.  Please point to an existing, complete con
fig file:\n\n\n  1. Via the command-line flag --kubeconfig\n  2. Via the KUBECONFIG environment variable\n  3. In your home directory as ~/.kube/config\n\nTo view or setup config directly
use the 'config' command.", "stderr_lines": ["error: Missing or incomplete configuration info.  Please point to an existing, complete config file:", "", "", "  1. Via the command-line flag
 --kubeconfig", "  2. Via the KUBECONFIG environment variable", "  3. In your home directory as ~/.kube/config", "", "To view or setup config directly use the 'config' command."], "stdout"
: "", "stdout_lines": []}
```
e.g. like in [ci_framework/roles/edpm_prepare/tasks/main.yml](https://github.com/openstack-k8s-operators/ci-framework/blob/main/ci_framework/roles/edpm_prepare/tasks/main.yml#L25)